hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestStreamingSource.scala

```diff
@@ -51,45 +53,46 @@ class TestStreamingSource extends StreamTest {
     withTempDir { inputDir =>
       val tablePath = s"${inputDir.getCanonicalPath}/test_cow_stream"
       HoodieTableMetaClient.withPropertyBuilder()
-          .setTableType(COPY_ON_WRITE)
-          .setTableName(getTableName(tablePath))
-          .setPayloadClassName(DataSourceWriteOptions.PAYLOAD_CLASS_NAME.defaultValue)
+        .setTableType(COPY_ON_WRITE)
+        .setTableName(getTableName(tablePath))
+        .setPayloadClassName(DataSourceWriteOptions.PAYLOAD_CLASS_NAME.defaultValue)
         .setPreCombineField("ts")
-          .initTable(spark.sessionState.newHadoopConf(), tablePath)
+        .initTable(spark.sessionState.newHadoopConf(), tablePath)
 
       addData(tablePath, Seq(("1", "a1", "10", "000")))
       val df = spark.readStream
         .format("org.apache.hudi")
+        .option(DataSourceReadOptions.READ_BY_STATE_TRANSITION_TIME.key(), useTransitionTime)
```

Comment:
this should be parameterized, otherwise, the default commit instant time is not test-covered

Reply:
By default `useTransitionTime` is false, and this test covers the default commit instant time, while `TestStreamSourceReadByStateTransitionTime` extends this class and override `useTransitionTime` to true.